### ISSUE TITLE
Use es2015+ features

### ISSUE THREAD

**Issue by deepnum** [brianc/node-postgres#2007]
The code still uses es5

**Comment by charmander** [comment]
No, the code isn’t ES5-compatible. It does contain things that existed in ES5; those continue to work. Is there some specific advantage you were hoping pg could expose with a new language feature?

**Comment by deepnum** [comment]
Like using Class instead of constructor function. You can use tools like lebab @lebab to automatically do this for you.
